storybook/src/components/form/form-group-radios.css-template.ts

```diff
@@ -31,8 +31,9 @@ export const cssFormGroupRadios: ComponentImplementation<FormGroupRadios<Templat
           aria-describedby=${ifDefined(ariaDescribedBy)}
           aria-errormessage=${ifDefined(ariaErrorMessage)}
         >
+          <legend class="sr-only">${formGroup.label}</legend>
```

Comment:
Misschien begrijp ik het niet helemaal goed, maar ipv dat de root fieldset en legend zijn verwijdert is er nu een sr-only legend bijgekomen.